src/session/session_compact.c

```diff
@@ -355,20 +355,7 @@ __compact_worker(WT_SESSION_IMPL *session)
                       session->op_handle[i]->name);
                 }
 
-                WT_STAT_CONN_INCR(session, session_table_compact_conflicting_checkpoint);
-
-                __wt_verbose_info(session, WT_VERB_COMPACT,
-                  "The compaction of the data handle %s returned EBUSY due to an in-progress "
-                  "conflicting checkpoint.%s",
-                  session->op_handle[i]->name,
-                  background_compaction ? "" : " Compaction of this data handle will be retried.");
-
                 ret = 0;
-
-                /* Don't retry in the case of background compaction, move on. */
-                if (!background_compaction)
```

Comment:
As discussed on Slack, we are modifying the behaviour of bg compaction if you remove this.